mne/io/tests/test_pick.py

```diff
@@ -11,6 +11,10 @@
 from mne.io.constants import FIFF
 from mne.datasets import testing
 from mne.forward.tests import test_forward
+from mne.io import Raw
+from mne import pick_info
+import mne
```

Comment:
We try to combine imports if possible, so whatever you use from `mne` namespace here can be imported on line `6` (or actually you'll need to expand that list onto the next line) and also line 15 can be combined with that list as well